### PR TITLE
Update innodb cluster job

### PR DESCRIPTION
**Pull request by kaushikpuneet07**

### CODE REVIEW

File: ps/jenkins/test-ps-innodb-cluster-parallel.groovy [hunk 1]
```diff
@@ -1,17 +1,18 @@
-library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+library changelog: false, identifier: 'lib@innodb', retriever: modernSCM([
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster-parallel.groovy:1]: should be fixed before merge

File: ps/jenkins/test-ps-innodb-cluster-parallel.groovy [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
     $class: 'GitSCMSource',
-    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+    remote: 'https://github.com/kaushikpuneet07/jenkins-pipelines.git'
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster-parallel.groovy:3]: should be fixed before merge

File: ps/jenkins/test-ps-innodb-cluster-parallel.yml [hunk 3]
```diff
@@ -12,9 +12,9 @@
     pipeline-scm:
         scm:
             - git:
-                url: https://github.com/Percona-Lab/jenkins-pipelines.git
+                url: https://github.com/kaushikpuneet07/jenkins-pipelines.git
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster-parallel.yml:15]: should be fixed before merge

File: ps/jenkins/test-ps-innodb-cluster-parallel.yml [hunk 4]
```diff
@@ -12,9 +12,9 @@
...
                 branches:
-                    - 'master'
+                    - 'innodb'
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster-parallel.yml:17]: should be fixed before merge

File: ps/jenkins/test-ps-innodb-cluster.groovy [hunk 5]
```diff
@@ -1,6 +1,6 @@
-library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+library changelog: false, identifier: 'lib@innodb', retriever: modernSCM([
     $class: 'GitSCMSource',
-    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+    remote: 'https://github.com/kaushikpuneet07/jenkins-pipelines.git'
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster.groovy:3]: should be fixed before merge

File: ps/jenkins/test-ps-innodb-cluster.groovy [hunk 6]
```diff
@@ -16,7 +16,7 @@ void installDependencies() {
 
     sh '''
         rm -rf package-testing
-        git clone https://github.com/Percona-QA/package-testing
+        git clone https://github.com/kaushikpuneet07/package-testing.git --branch innodb
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster.groovy:19]: should be fixed before merge

File: ps/jenkins/test-ps-innodb-cluster.groovy [hunk 7]
```diff
@@ -38,6 +38,7 @@ void runMoleculeAction(String action, String scenario) {
     withCredentials(awsCredentials) {
         sh """
             source venv/bin/activate
+            cat  package-testing/tasks/test_prep.yml
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster.groovy:41]: why do we need it?

File: ps/jenkins/test-ps-innodb-cluster.yml [hunk 8]
```diff
@@ -14,9 +14,9 @@
     pipeline-scm:
         scm:
             - git:
-                url: https://github.com/Percona-Lab/jenkins-pipelines.git
+                url: https://github.com/kaushikpuneet07/jenkins-pipelines.git
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster.yml:17]: should be fixed before merge

File: ps/jenkins/test-ps-innodb-cluster.yml [hunk 9]
```diff
@@ -14,9 +14,9 @@
...
                 branches:
-                    - 'master'
+                    - 'innodb'
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster.yml:19]: should be fixed before merge

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Review by panchal-yash — APPROVED** [review]

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**